src/helix-ui/elements/HXSearchAssistanceElement.js

```diff
@@ -36,14 +36,14 @@ export class HXSearchAssistanceElement extends HXElement {
         return [ 'open' ];
     }
 
-    attributeChangedCallback (attr, oldVal, newVal) {
+    $onAttributeChange (attr, oldVal, newVal) {
         let isOpen = (newVal !== null);
         if (newVal !== oldVal) {
             this.$emit(isOpen ? 'open' : 'close');
         }
-    }
+    }//$onAttributeChange()
 
-    connectedCallback () {
+    $onConnect () {
```

Comment:
The order of methods here is different than the other files. I'd keep the order consistent (so `$onConnect` would go before `$onAttributeChange`).